### PR TITLE
Add info panel advising inclusion of Spring Security in Thymeleaf projects

### PR DESCRIPTION
**Pull request by JoelClemence**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: README.md [hunk 1]
```diff
@@ -86,4 +105,4 @@ Fragment that contains several links and information for the user. Links to Your
 
 ### error.html
 
-Generic error page that gives the user an option to email Companies House. For this to work there must be a ```baseLayout.html``` file within the ```layouts/```` directory.
\ No newline at end of file
+Generic error page that gives the user an option to email Companies House. For this to work there must be a ```baseLayout.html``` file within the ```layouts/```` directory.
```
Comment (ch-tstrickland) [inline README.md:108]: What's changed here?

Comment (JoelClemence) [reply README.md:108]: New line I am guessing

File: README.md [hunk 2]
```diff
@@ -22,6 +22,25 @@ Things To consider
 
 + Please use this [Pipeline](https://ci-platform.companieshouse.gov.uk/teams/team-development/pipelines/common-web-java) and make sure respective `source-code-main` or `source-code-main-8` task gets started once the PR is created or after the PR is merged to `main` or `main-8` and once the pipeline tasks are complete then use the created tags respectively.
 
+> [!NOTE]
+> When using in a web application remember to include a dependency on Spring
```
Comment (ch-tstrickland) [inline README.md:26]: Perhaps more forceful?
'remember to'
->
'you MUST'

File: README.md (hunk 1 above)
Comment (JoelClemence) [reply README.md:108]: I think my editor adds them into end of file automatically

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: README.md (hunk 1 above)
Comment (ch-tstrickland) [reply README.md:108]: Naughty editor.

**Review by ch-tstrickland — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by JoelClemence** [comment]
There is going to be more to do - will update following the CSRF has been mitigated with a scalable solution.

**Pull request closed without merging**